Install returns error code 255 in CI/CD

When building my project via Github Actions, the install fails with error code 255 returned. I set up an example [in this repo](https://github.com/BenBenBenB/godot-cicd-test).

I am running `godot --headless -s plug.gd install production` in [the workflow](https://github.com/BenBenBenB/godot-cicd-test/blob/main/.github/workflows/godot-ci.yml#L22) which is running on Ubuntu.

Example output from a failed workflow: https://github.com/BenBenBenB/godot-cicd-test/actions/runs/10549781403/job/29225223972
```
Run godot --headless -s plug.gd install production
Godot Engine v4.3.stable.official.77dcf97d8 - https://godotengine.org/

Installing...
Installing plugin Godot-StateMachineNodes...
Downloading Godot-StateMachineNodes from ***github.com/ninstar/Godot-StateMachineNodes.git...
Successfully download Godot-StateMachineNodes
Installing files for Godot-StateMachineNodes...
Installed 9 files for Godot-StateMachineNodes
Finished, elapsed 0.692s
Error: Process completed with exit code 255.
```
I am assuming a -1 is returned and cast as an unsigned int8 to become 255.

I made it work in my projects by adding `request_quit(0)` to the end of `_finalize()`, but I feel there may be a more elegant way to exit execution.
- The change: https://github.com/BenBenBenB/godot-cicd-test/commit/ddcbdda788f0903256f5f8570f0c00ce36461ada
- The successful pipeline: https://github.com/BenBenBenB/godot-cicd-test/actions/runs/10549817571

--- a/addons/gd-plug/plug.gd
+++ b/addons/gd-plug/plug.gd
@@ -98,7 +98,7 @@ func _initialize():
 				logger.error("Unknown command %s" % args[0])
 				show_syntax()
 	# NOTE: Do no put anything after this line except request_quit(), as _plug_*() may call request_quit()
-	request_quit()
+	request_quit(0)
 
 func show_syntax():
 	logger.info("gd-plug - Minimal plugin manager for Godot")

